tests/src/providers/testqgsogrprovider.cpp

```diff
@@ -184,11 +184,8 @@ void failOnWarning( QtMsgType type, const QMessageLogContext &context, const QSt
 
 void TestQgsOgrProvider::testThread()
 {
-  // Disabled by @m-kuhn
   // This test is flaky
   // See https://travis-ci.org/qgis/QGIS/jobs/505008602#L6464-L7108
-  if ( !QgsTest::runFlakyTests() )
```

Comment:
Don't we still want this granular control?

Reply:
yep, I forgot it was the test within the test.